side_hustle/settings.py

```diff
@@ -37,7 +37,7 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
-    'jobboard.apps.JobboardConfig',
```

Comment:
This changes is related to removing jobboard app commit, isnt it?

Reply:
right.

Reply:
Please move it to the first commit